templates/component-pages/component-name/accessibility.mdx

```diff
@@ -39,3 +48,22 @@ You can have as many Keyboard control blocks as you need.
 
 </KeyboardControl>
 </KeyboardControls>
```

Comment:
What was onerous with Carousel was having to verbalise user journeys for focus. It must be horrible to read.
How can we describe focusable user journeys better?

Could we annotate the elements in a view and then call them out as action &  effect?